src/View/TemplatePaths.php

```diff
@@ -258,9 +258,11 @@ public function resolveTemplateFileForControllerAndActionAndFormat($controller,
         $controller = str_replace('\\', '/', $controller);
         $action = ucfirst($action);
         $identifier = $controller . '/' . $action . '.' . $format;
+        $identifier = ltrim($identifier, '/');
```

Comment:
Is there a specific reason for the separate line here but combined line below?

Reply:
Thanks for the fast review. No it isn't. But I can adapt this of course. Which variant do you prefer?

Reply:
IMO this can be merged here, there is no real need for debugging interim values here.

Reply:
Thanks, I took care of that